query/QueryManager.java

```diff
@@ -100,4 +160,11 @@ public QueryManager(RPCTransaction rpcTransaction) {
                 .setQueryReq(request.setOptions(options(options)));
         return rpcTransaction.stream(req, responseReader);
     }
+
+    private Concept concept(ConceptProto.Concept owner) {
```

Comment:
For consistency with other `Concept` deserializers, this method should be `static`, named `of`, and live in `Concept.java`.

Reply:
Should we put the method in the `grakn.client.concept.Concept` interface? or should we create another `Concept` class in `grakn.client.concept.answer.Concept`?

I am inclined for the former, but it is an `interface` and adding a conversion method that knows about the implementation detail there .

Reply:
We have a `ConceptImpl` class now! Pull from `master` if you don't have in your codebase yet.

Reply:
cool!

Reply:
Addressed in https://github.com/graknlabs/client-java/pull/218/commits/a1a66fd1d280199b60bed6f2ecb2e9c0ee4538d1